json-schemas/interface-methods/protocol-rule-set.json

```diff
@@ -106,10 +106,54 @@
           "minimum": 0
         }
       }
+    },
+    "$tags": {
+      "type": "object",
+      "minProperties": 1,
+      "patternProperties": {
+        "requiredTags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "allowUndefinedTags": {
```

Comment:
For my understanding: what is a use case for this property?

Reply:
To give the option for giving rules around specific necessary tags, as well as allowing open-ended tagging for some use-cases, figured it would give some flexibility there.